### PR TITLE
Pipenv support

### PR DESCRIPTION
**Pull request by eladiw**

### CODE REVIEW

File: docs/pipenv_readme.md [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+
+### 3. Install all requirements in the Pipfile, including dev requirements
+
+```
+$ pipenv install --dev --sequential
+```
+
+### 4. Run all tests
+```
+$ pipenv run pytest
```
Comment (itye-msft) [inline docs/pipenv_readme.md]: After pipenv shell, the environment is activated, and no need to use `pipenv`. You can just call `pytest` inside the virtual environment.

File: docs/pipenv_readme.md [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+$ brew install pipenv
+```
+
+Additional installation instructions: https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv
+
+
+### 2. Create virtualenv for the project
+From the python project's root folder, run:
+```
+$ pipenv shell
```
Comment (itye-msft) [inline docs/pipenv_readme.md]: Also explain how to deactivate the environment.

File: docs/pipenv_readme.md (hunk 1 above)
Comment (itye-msft) [reply docs/pipenv_readme.md]: Same remark for other pipenv in the rest of the file

Comment (eladiw) [reply docs/pipenv_readme.md]: @omri is it more accurate to switch between 2 and 3. meaning first installing then entering the shell?

Comment (eladiw) [reply docs/pipenv_readme.md]: fixed

File: docs/pipenv_readme.md (hunk 2 above)
Comment (eladiw) [reply docs/pipenv_readme.md]: fixed (removed the shell command, so no need for deactivation)

Comment (itye-msft) [reply docs/pipenv_readme.md]:
```suggestion
$ pipenv shell
```

To Deactivate your virtual environment, run:
```
deactivate
```

File: docs/pipenv_readme.md [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+```
+#### Homebrew
+```
+$ brew install pipenv
+```
+
+Additional installation instructions: https://pipenv.readthedocs.io/en/latest/install/#installing-pipenv
+
+
+### 2. Create virtualenv for the project
```
Comment (itye-msft) [inline docs/pipenv_readme.md]:
```suggestion
### 2. Create virtualenv for the project (for local development)
```

**Comment by omri374** [comment]
@eladiw IMHO we're still missing some consistency and structure with the docs. The development.md should be combined with the pipenv_readme.md. They currently say two different things and the pipenv_readme is not visible enough. 
Other than that it looks great.

**Pull request merged**